fMRIVolume/scripts/IntensityNormalization.sh

```diff
@@ -132,11 +155,28 @@ PctCoverage=`echo "scale=4; 100 * ${NvoxFinalMask} / ${NvoxBrainMask}" | bc -l`
 echo "PctCoverage, NvoxFinalMask, NvoxBrainMask" >| ${FinalMask}.stats.txt
 echo "${PctCoverage}, ${NvoxFinalMask}, ${NvoxBrainMask}" >> ${FinalMask}.stats.txt
 
+# If a user requested, use a T1 mas, a dilated T1 mask or no mask instedad when generating final BOLD timeseries
+
+if [ "${UseMask}" = "DILATEDT1" ] ; then
+    # -- create a dilated version of T1 mask
+    ${FSLDIR}/bin/fslmaths ${BrainMask} -dilF ${BrainMask}_dil
```

Comment:
Is a single round of dilation enough to accomplish your purposes with the `DILATEDT1` mask?  Seems like you'd want more than that...

Reply:
The options now will be 

* `T1_fMRI_FOV`: combined T1w brain mask and FOV masks (the default), 
* `T1_DILATED_fMRI_FOV`: a once dilated T1w brain based mask combined with fMRI FOV
* `T1_DILATED2_fMRI_FOV`: a twice dilated T1w brain based mask combined with fMRI FOV, 
* `fMRI_FOV`: a fMRI FOV mask

Reply:
I have also changed the flow of the code so that the `FinalMask` is the mask specified. And that the statistics are reported both on the FOV of the T1w brain as well as the actual mask used, with the mask used also listed in the ${FinalMask}.stats.txt